source/reference/beta-program.txt

```diff
@@ -94,9 +91,8 @@ Beta Features
    :titlesonly:
 
    /reference/beta-program/abc-migration
-   /reference/beta-program/namespace-remapping
+   /reference/beta-program/many-to-one
    /reference/beta-program/document-filtering
-   /reference/beta-program/orr 
    /reference/beta-program/destinationDataHandling 
-   /reference/beta-program/migration-name
```

Comment:
That makes sense — I initially kept it in there since at the end of the day it *is* still a new option in the 1.8 beta, even if it isn't a formal feature.